Framework/Algorithms/src/SetUncertainties.cpp

```diff
@@ -68,19 +102,26 @@ void SetUncertainties::exec() {
   for (int64_t i = 0; i < int64_t(numHists); ++i) {
     PARALLEL_START_INTERUPT_REGION
 
```

Comment:
Skip processing if spectra is masked, otherwise you can have masked spectra with intensity 0, and error 1